jenkins/job-configs/kubernetes-jenkins/bootstrap-ci.yaml

```diff
@@ -251,6 +251,11 @@
         timeout: 70
         frequency: 'H/5 * * * *' # At least every 30m
         trigger-job: 'ci-kubernetes-build'
+    - kubernetes-e2e-gci-memcg-serial:
```

Comment:
Move this to under the section of https://github.com/kubernetes/test-infra/blob/master/jenkins/job-configs/kubernetes-jenkins/bootstrap-ci-repo.yaml#L136, rename to `ci-kubernetes-node-memcg-serial`